pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -159,8 +177,9 @@ func initialConditions(defaults []types.Condition) []types.Condition {
 	conditions := make([]types.Condition, len(defaults))
 	copy(conditions, defaults)
 	for i := range conditions {
-		// TODO(random-liu): Validate default conditions
-		conditions[i].Status = false
+		if conditions[i].Status == "" {
```

Comment:
IIUC, by changing to this, NPD will allow user to custom the initial `status` field value.

Reply:
Hm, let's not do that.

Reply:
Done